mu4e/mu4e-window.el

```diff
@@ -267,6 +265,7 @@ for BUFFER-OR-NAME to be displayed in."
             ('(view . vertical) '((window-min-width . fit-window-to-buffer)))
             (`(,_ . t) nil)))
          (window-action (cond
+                         ((eq buffer-type 'main) '(display-buffer-full-frame))
                          ((and (eq buffer-type 'compose) mu4e-compose-in-new-frame)
```

Comment:
Note, `display-buffer-full-frame` is emacs-29, but we should support emacs 26+... what can we use instead?

Reply:
Backport it maybe:

```
(defun display-buffer-full-frame (buffer alist)
  "Display BUFFER in the current frame, taking the entire frame.
ALIST is an association list of action symbols and values.  See
Info node `(elisp) Buffer Display Action Alists' for details of
such alists.

This is an action function for buffer display, see Info
node `(elisp) Buffer Display Action Functions'.  It should be
called only by `display-buffer' or a function directly or
indirectly called by the latter."
  (when-let ((window (or (display-buffer-reuse-window buffer alist)
                         (display-buffer-same-window buffer alist)
                         (display-buffer-pop-up-window buffer alist)
                         (display-buffer-use-some-window buffer alist))))
    (delete-other-windows window)
    window))
```

Reply:
Yeah, guess that's the best way. Pushed; thanks.

Reply:
Yeah it's not perfect but the code is simple enough and it preserves the behaviour.